ocs_ci/ocs/platform_nodes.py

```diff
@@ -397,16 +397,17 @@ def stop_nodes(self, nodes, wait=True):
         )
         self.aws.stop_ec2_instances(instances=instances, wait=wait)
 
-    def start_nodes(self, nodes, wait=True):
+    def start_nodes(self, instances, nodes, wait=True):
```

Comment:
I think there are a few places where this signature change isn't being considered:

![Screen Shot 2020-05-14 at 12 27 16 PM](https://user-images.githubusercontent.com/1605721/81977108-6780e400-95de-11ea-8adb-500706d2a92a.png)

Reply:
the only reference i see is from test_monitoring_on_negative_scenario.py, last time i did fix the place where the signature was not using keyword arg, all tests in z_cluster passed in jenkins run. the only new one i see is at test_monitoring.

Reply:
The PR test was configured to run tier1. `test_automated_recovery_from_failed_nodes_reactive_IPI` is `tier4b` and `test_nodes_restart` is `tier4a`

Reply:
This is good point @clacroix12 we should probably report back here to github with what markers the test PR validation was executed. WDYT? Can you open the infra ticket for it please?

Reply:
the reason why ```test_automated_recovery_from_failed_nodes_reactive_IPI`` is missed was it was merged after my PR, the ```test_nodes_restart``` uses kw args so no issue there.

Reply:
@vasukulkarni You are adding a new parameter with no default value. Existing calls to this function will fail.

Reply:
@petr-balogh https://projects.engineering.redhat.com/browse/OCSQE-342

Reply:
@clacroix12 please see the diff ```instances = instances or self.get_ec2_instances(nodes)```

Reply:
heh, I was assuming they are kw args, which is not the case, thanks!

Reply:
Fixed in: e6e1d92be2ab6d0394125839dbb091dfd2422b61